website/docs/internals/json-format.html.md

```diff
@@ -101,6 +101,12 @@ For ease of consumption by callers, the plan representation includes a partial r
       // representation
       "address": "module.child.aws_instance.foo[0]",
 
+      // "previous_address" is the full absolute address of this resource
+      // instance as of the previously completed Terraform run, and is only
+      // present if a configuration-driven move operation is planned for this
```

Comment:
So far "configuration-driven move" was just a project name rather than a user-facing feature name. Elsewhere I tried to keep most docs generic and just say "refactoring" in anticipation of other features coming later, but this particular thing does seem like it's specific to `moved` blocks and so it would probably be weird to over-generalize it.

With that said, here's an attempt at a different phrasing that tries to avoid turning our project name into part of the user model:

```
    // "previous_address" is the absolute address of this resource
    // instance as it was known after the previously Terraform run.
    // Included only if the address has changed, e.g. by handling
    // a "moved" block in the configuration.
```

The "e.g." here is trying to allow for the "implied moves" of #29605, and possibly other things we might add to our refactoring toolset in the future, while focusing on the main case.

(This isn't the first time I've wished for the ability to put a link inside one of these comments, so that we could more directly connect what we're talking about with the main sections describing the features; hopefully we can find a nice way to achieve that in a future iteration of the design of these docs, but that's a task for another day.)

Reply:
Thanks! I updated the comment.